opus/import/populate_obs_instrument_VGUVS_prof.py

```diff
@@ -0,0 +1,611 @@
+    return populate_occ_ra_dec_helper_index(**kwargs)[1]
+
+def populate_obs_general_VGUVS_declination1_PROF(**kwargs):
+    return populate_occ_ra_dec_helper_index(**kwargs)[2]
+
+def populate_obs_general_VGUVS_declination2_PROF(**kwargs):
+    return populate_occ_ra_dec_helper_index(**kwargs)[3]
+
+
+### OBS_TYPE_IMAGE TABLE ###
+
+def populate_obs_type_image_VGUVS_image_type_id_PROF(**kwargs):
+    return None
+
+def populate_obs_type_image_VGUVS_duration_PROF(**kwargs):
+    return None
+
+def populate_obs_type_image_VGUVS_levels_PROF(**kwargs):
+    return None
+
+def populate_obs_type_image_VGUVS_lesser_pixel_size_PROF(**kwargs):
+    return None
+
+def populate_obs_type_image_VGUVS_greater_pixel_size_PROF(**kwargs):
+    return None
+
+
+### OBS_WAVELENGTH TABLE ###
+
+def populate_obs_wavelength_VGUVS_wavelength1_PROF(**kwargs):
+    metadata = kwargs['metadata']
+    index_row = metadata['index_row']
+    wl1 = index_row['MINIMUM_WAVELENGTH']
+    return wl1
+
+def populate_obs_wavelength_VGUVS_wavelength2_PROF(**kwargs):
+    metadata = kwargs['metadata']
+    index_row = metadata['index_row']
+    wl2 = index_row['MAXIMUM_WAVELENGTH']
+
+    return wl2
+
+def _wave_res_helper(**kwargs):
+    metadata = kwargs['metadata']
+    wl_row = metadata['obs_wavelength_row']
+    wl1 = wl_row['wavelength1']
+    wl2 = wl_row['wavelength2']
+    if wl1 is None or wl2 is None:
+        return None
+    return wl2 - wl1
+
+def populate_obs_wavelength_VGUVS_wave_res1_PROF(**kwargs):
+    return _wave_res_helper(**kwargs)
+
+def populate_obs_wavelength_VGUVS_wave_res2_PROF(**kwargs):
+    return _wave_res_helper(**kwargs)
+
+def populate_obs_wavelength_VGUVS_wave_no1_PROF(**kwargs):
+    metadata = kwargs['metadata']
+    wavelength_row = metadata['obs_wavelength_row']
+    wl2 = wavelength_row['wavelength2']
+    if wl2 is None:
+        return None
+    return 10000 / wl2 # cm^-1
+
+def populate_obs_wavelength_VGUVS_wave_no2_PROF(**kwargs):
+    metadata = kwargs['metadata']
+    wavelength_row = metadata['obs_wavelength_row']
+    wl1 = wavelength_row['wavelength1']
+    if wl1 is None:
+        return None
+    return 10000 / wl1 # cm^-1
+
+# Same logic as wave_res
+def _wave_no_res_helper(**kwargs):
+    metadata = kwargs['metadata']
+    wl_row = metadata['obs_wavelength_row']
+    wno1 = wl_row['wave_no1']
+    wno2 = wl_row['wave_no2']
+    if wno1 is None or wno2 is None:
+        return None
+    return wno2 - wno1
+
+def populate_obs_wavelength_VGUVS_wave_no_res1_PROF(**kwargs):
+    return _wave_no_res_helper(**kwargs)
+
+def populate_obs_wavelength_VGUVS_wave_no_res2_PROF(**kwargs):
+    return _wave_no_res_helper(**kwargs)
+
+def populate_obs_wavelength_VGUVS_spec_flag_PROF(**kwargs):
+    return 'N'
+
+def populate_obs_wavelength_VGUVS_spec_size_PROF(**kwargs):
+    return 1
+
+def populate_obs_wavelength_VGUVS_polarization_type_PROF(**kwargs):
+    return 'NONE'
+
+
+### populate_obs_occultation TABLE ###
```

Comment:
This comment was previously wrong in all the files - can you fix it? It should say "### OBS_OCCULTATION TABLE ###"